src/Servers/IIS/AspNetCoreModuleV2/Microsoft.AspNetCore.AspNetCoreModuleV2/Directory.Build.targets

```diff
@@ -21,14 +21,15 @@
     <TargetPath>$(PackageOutputPath)$(PackageId).$(PackageVersion).nupkg</TargetPath>
   </PropertyGroup>
 
-  <Target Name="GetArtifactInfo" Returns="@(ArtifactInfo)" Condition=" '$(IsPackable)' != 'false' ">
+  <Target Name="GetArtifactInfo" Returns="@(ArtifactInfo)">
     <ItemGroup>
       <ArtifactInfo Include="$(TargetPath)">
         <ArtifactType>NuGetPackage</ArtifactType>
         <PackageId>$(PackageId)</PackageId>
-        <Version>$(PackageVersion)</Version>
-        <Certificate>$(PackageSigningCertName)</Certificate>
-        <ShouldBeSigned>true</ShouldBeSigned>
+        <Version Condition="'$(IsPackable)' == 'true'">$(PackageVersion)</Version>
```

Comment:
FYI @mikaelm12 - this might need to be backported to 2.1 as well. I think it fixes this error:

```
2019-01-11T18:16:22.2666869Z D:\a\1\s\build\repo.targets(158,5): error : Could not detect version information for package id: [D:\a\.dotnet\buildtools\korebuild\2.2.1-build-20190110.1\KoreBuild.proj]
2019-01-11T18:16:22.2678259Z D:\a\1\s\build\repo.targets(158,5): error :  * Microsoft.AspNetCore.AspNetCoreModule [D:\a\.dotnet\buildtools\korebuild\2.2.1-build-20190110.1\KoreBuild.proj]
2019-01-11T18:16:22.2678799Z D:\a\1\s\build\repo.targets(158,5): error :  * Microsoft.AspNetCore.AspNetCoreModuleV2 [D:\a\.dotnet\buildtools\korebuild\2.2.1-build-20190110.1\KoreBuild.proj]
2019-01-11T18:16:22.2679135Z     4 Warning(s)
```